File: dart/lib/model/main_user_stats_response.dart
part of swagger.api;

class MainUserStatsResponse {
  
  int numPins = null;
  

  int totalSize = null;
  
  MainUserStatsResponse();

  @override
  String toString() {
    return 'MainUserStatsResponse[numPins=$numPins, totalSize=$totalSize, ]';
  }

  MainUserStatsResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    numPins =
        json['numPins']
    ;
    totalSize =
        json['totalSize']
    ;
  }

  Map<String, dynamic> toJson() {
    return {
      'numPins': numPins,
      'totalSize': totalSize
     };
  }

  static List<MainUserStatsResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<MainUserStatsResponse>() : json.map((value) => new MainUserStatsResponse.fromJson(value)).toList();
  }

  static Map<String, MainUserStatsResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MainUserStatsResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MainUserStatsResponse.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/estuary_client/Model/MainUserStatsResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace estuary_client.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class MainUserStatsResponse {
    /// <summary>
    /// Gets or Sets NumPins
    /// </summary>
    [DataMember(Name="numPins", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "numPins")]
    public int? NumPins { get; set; }

    /// <summary>
    /// Gets or Sets TotalSize
    /// </summary>
    [DataMember(Name="totalSize", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "totalSize")]
    public int? TotalSize { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MainUserStatsResponse {\n");
      sb.Append("  NumPins: ").Append(NumPins).Append("\n");
      sb.Append("  TotalSize: ").Append(TotalSize).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
